Add PyPI publish docs to project_setup.md (#170)
NLeSC/python-template: {{cookiecutter.project_slug}}/project_setup.md

-   Don't forget to update the version number before [making a
    release](https://guide.esciencecenter.nl/#/best_practices/releases)!

## Publish on Python Package Index (PyPI)

> To make the publishing to PyPI you need to create a [PyPI API token](https://pypi.org/help#apitoken) and

Reviewer: ```suggestion
To publish your package on PyPI, you need to create a [PyPI API token](https://pypi.org/help#apitoken) and
```
